docs/guide.md

```diff
@@ -261,6 +261,8 @@ Make sure to substitute `me@example.com` with your actual email.
 
 Visit `https://<your-domain-name>` to access `code-server`. Congratulations!
 
+Note: if you set `proxy_set_head Host $host;` in your reverse proxy config, it will change the address displayed in the green section of code-server in the bottom left to show the correct address.
```

Comment:
```suggestion
Note: if you set `proxy_set_header Host $host;` in your reverse proxy config, it will change the address displayed in the green section of code-server in the bottom left to show the correct address.
```